WTWSMS/common/landed_titles/WtWSMS_pagan_religious_heads.txt

```diff
@@ -632,7 +632,7 @@ d_jewish = {
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
 }
 
-d_melchisedechianism = {
+k_melchisedechianism = {
```

Comment:
> Melchisedech's title was the "King-Priest of Salem". All Biblical concordance argues he was a king. Father Abraham, when he gave him tribute, treated him as such. In addition, a king-tier Melchisedechianism allows duchies under it. All Biblical references to it are king-level. When the Bible says that Jesus is a priest "in the order of Melchisidech", it's used alongside His (and God's) title of King of Kings and Lord of Lords. It has "KINGDOM" in the name. All historical records, including the Bible, call it a kingdom.

You know that other realms are referred and called as "kingdoms" but get to be petty kingdoms? Why not here?

> The burden of proof is on YOU as to why it should be a duchy.

That is a logical fallacy, just so you know it.